packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -6,368 +6,764 @@
     <form
         data-ng-if="!$ctrl.isFetchingCreationOptions"
         name="$ctrl.newAccountForm"
+        class="container m-0"
```

Comment:
**question:** Why using `m-0`? (`container` should keep its margins (auto)).

Reply:
Because container has auto right and left marge.
So the whole content go to center page.

Reply:
Did you tested it with `container-fluid` then?

Reply:
Yep i just test it, the container took much more page width.
For a large screen is lot.

Container is good.